CHANGELOG.md

```diff
@@ -3,6 +3,14 @@ All notable changes to this project 1.4.x per each release will be documented in
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [v1.4.0.7] - 2020-05-25
+
+### Added
+- Open Banking SG specification support for OB 1.4.0
+
+For detailed information on the tasks carried out during this release, please refer,
```

Comment:
@Akila94 let us remove this line and just add to refer issue and issue link.

e.g. Open Banking SG specification support for OB 1.4.0 (refer issue https://github.com/wso2/docker-open-banking/issues/68).

Reply:
Updated.